src/interaction/MeasureLength.js

```diff
@@ -49,15 +73,74 @@ olBase.inherits(exports, ngeoInteractionMeasure);
 /**
  * @inheritDoc
  */
-exports.prototype.createDrawInteraction = function(style, source) {
+exports.prototype.createDrawInteraction = function(style) {
   return new olInteractionDraw({
     type: /** @type {ol.geom.GeometryType} */ ('LineString'),
-    source: source,
+    geometryFunction: (...args) =>
+      this.linestringGeometryFunction(...args),
```

Comment:
You should use bind instead of the `...args` magics
`this.linestringGeometryFunction.bind(this)`